case-studies/docker/docker-case-study.md

```diff
@@ -0,0 +1,265 @@
+
+
+
+## Brief History 
+
+when docker launched in 2013 it’s, was a Linux-based kernel containerization technology. Only Linux user was using docker before docker partner up with [Redhat](https://www.redhat.com/en) and This may be the most prosperous partnership Docker has made because this led to even bigger partnership in docker history, [Microsoft](https://www.microsoft.com/) wants to make this technology ready for Windows users and Windows server. Before Microsoft made partnership with Docker [LinkedIn](https://www.linkedin.com/) so that Docker had grew by 160% present by 2016. By 2019 docker and Microsoft released docker to windows and windows server.  
+
+
+
+- 2013 Docker and Red Hat Announced a collaboration around Fedora, Red Hat Enterprise, OpenShift
+- 2014 October Microsoft announced the intergration of the Docker engine into Windows Server, Support for Docker client in windows. 
+- November 2014: Docker container services ware announced for the Amazon elastic Computer cloud. 
+- 2014 November: Docker annonced a partership with Stratoscal.
+- 2014 December IBM anounced a partnershipt with Docker that enables Docker to integrate more to IBM cloud.
+- 2015 December: Oracle Cloud added Docker container support after acquiring SlackEngine.  
+- 2016 April: Windoc and independent software vendor released a port of open source project to windows, supporting windows 2012 R2 
+- 2016 May: Analysis of Linkedin profile mentoions showed Docker grew by 160% in 2016
+- 2019 May Announced the second verison of windows subsystem for link (WSL) from microsoft
+- 2019 aug Annoced a backport of WLS2 to windows from microsoft 
+- 2021 Aug: Docker Desktop from windows and MacOS is no longer fre for enterprise user 
+
+## Features 
+
+What is Docker the term docker can refer to Docker engine or to the company Docker Inc. Docker is Containerization technology around Docker engine, but we are going to this cover what is underneath Docker envierment
+
+**Features**
+
+* Docker Engine 
+* Docker Hub
+* Docker Image 
+* Docker Container 
+    
+
+### Docker Engine 
+
+What is Docker engine is Client-server Technology this support the tasks and workflow that is involved in building, shipping and running the containerizing technology. 
+
+Docker Engine using a server with daemon ##Link to deamon# process technology called dockerd using REST-API and a client-side CLI ## Link to CLI## (Command-line Interface) called docker.
```

Comment:
##Link to deamon#

What is this? I don't recognise it as markdown syntax.

Reply:
This wan was for me to add a link to information about the daemon link and link for CLI.